CDCgov/prime-reportstream
# Adds enrichments

This PR adds the ability to run enrichments on a bundle. 

Test Steps:
1. Make sure the unit tests and integration tests pass.
2. Run enrichments on a bundle via the command line. 

## Changes
- Allows enrichments on a bundle
- Adds the ability to add enrichments when running on the CLI
- Adds the ability to add enrichments to integration tests. 

## Checklist

### Testing

## Linked Issues
- #issue 12758
